### PR TITLE
Document wsod fix for XMLSiteMap Module

### PR DESCRIPTION
**Pull request by rachelwhitton**
@ari-gold thoughts?
cc @nataliejeremy for copy edit

### CODE REVIEW

**Comment by rachelwhitton** [comment]
cc @joshkoenig

File: source/docs/articles/sites/code/bots-and-indexing.md [hunk 1]
```diff
@@ -45,17 +45,45 @@ Disallow: /
...
-    $subject = $_SERVER['HTTP_HOST'];
-    $pattern = '#/pantheonsite.io/sitemap.xml$/#';
-    preg_match($pattern, $subject, $matches);
-    $counted = count($matches);
-    if (($counted > 0 ) && (php_sapi_name() != "cli")) {
-        header('HTTP/1.0 301 Moved Permanently');
-        header('Location: http://yoursite.com/sitemap.xml'. $_SERVER['REQUEST_URI']);
-        exit();
-    }
+You can index your site under your production domain. There are many contrib module options available for creating sitemaps for Drupal, including [XMLSiteMap](https://drupal.org/project/xmlsitemap) and [Site_Map](https://drupal.org/project/site_map). WordPress users can install the [Google XML Sitemaps](http://wpcrux.com/collectives/wordpress-xml-sitemap-plugins/) plugin, which will maintain sitemap updates automatically once the initial build has been completed. It is up to you to configure the extensions to work as you desire. Pantheon does not offer support for Drupal modules or WordPress plugins.
```
Comment (joshkoenig) [inline source/docs/articles/sites/code/bots-and-indexing.md]: Seems like we should mention Yost up here if we do at the bottom.

File: source/docs/articles/sites/code/bots-and-indexing.md [hunk 2]
```diff
@@ -45,17 +45,45 @@ Disallow: /
...
+### Troubleshooting
+
+#### Sitemaps Produce a White Screen of Death (WSOD)
+Some modules or plugins are configured by default to fetch all URLs at once during sitemap generation which can result in a blank white page (WSOD) due to exceeding PHP's memory limit. To resolve this issue, adjust the plugin or module configuration so that URLs are fetched individually instead of all at once.
+
+For example, if you're running the [XMLSiteMap](https://drupal.org/project/xmlsitemap) module (Drupal sites), navigate to `admin/config/search/xmlsitemap/settings` and uncheck **Prefetch URL aliases during sitemap generation**. Save the configuration and clear caches for the Live environment on the Pantheon Dashboard or via [Terminus](/docs/articles/local/cli): `terminus site clear-cache`
+
+Props to [Will Hall](https://twitter.com/HN_Will) for highlighting this solution in a related [blog post](http://www.willhallonline.co.uk/blog/get-xml-sitemaps-working-pantheon).
+
+#### Redirecting Sitemap Paths
```
Comment (joshkoenig) [inline source/docs/articles/sites/code/bots-and-indexing.md]: We should also point out that you can/should submit `sitemap_index.xml` directly to google webmaster tools. You only need to redirect if there are legacy sitemap submissions out there generating 404s.

**Comment by rachelwhitton** [comment]
Thanks @joshkoenig updated! cc @ari-gold 

@nataliejeremy can you review?

**Pull request merged**